governance/roles.md

```diff
@@ -103,23 +104,47 @@ technical leads.
 
 ## Role of project leads
 
-Project Leads will lead larger streams of work that require sustained
-effort and coordination.  In some cases, these projects remain ongoing.
+Project Leads lead specific projects where a deliverable has been defined 
+within the issue.
 
 Project Leads are nominated and approved by the following process:
-  1. Project Lead actively participates in the group, making a proposal or
-  volunteering to take on a project that has been prioritized by the group
-  1. A Chair or TL nominates a candidate.
+  1. Project Lead actively participates in the group, initiates the proposal or
+  volunteers to take on a project that has been prioritized by the group
+  1. A Chair or TL nominates a candidate, or requests the community to nominate in a slack thread
   1. The nomination is communicated via a pull request annotating the list of members in the
-  [TAG README](/README.md) with a link to the issue tracking the project.
+  [TAG README](/README.md) with a link to the issue tracking the project or on the issue tracking the project if a README is not yet made.
   The nomination is typically open for a week (but may be shorter with LGTM of at least two Chairs).
-  1. Members are encouraged to review the tracking issue and work together
-  to ensure that the Project Lead is set up for success or suggest alternatives.
+  1. The issue is updated with the assignment of the project lead as "assignee" alongside the TAG Leadership member.
+  1. Project Leads will be given the OWNER role of the directory or sub-directory for their team's activities in order to empower merge/approve.
+  
+  ### On-going projects
+  
+  On the occasion where a project becomes long term, Project Leads are also given the OWNER role of the directory or sub-directory along with 1 other nominated individual to ensure continuity in review and ongoing support of the project.
 
 Depending on the expected length of the project, Project Leads may be assigned
 as OWNERS of the project directory in this repository to help facilitate and moderate new content to the
 project.
 
+## Role of Team Leads
+
+Team Leads lead larger streams of work that require sustained effort and 
+coordination and remain ongoing.
+
+Teams must have a Co-Chair sponsor to maintain visibility in the team's work and to ensure it is progressing.  The Co-Chair sponsor also provides review and approval as part of the process whether be on engagement or deliverable publishing (such as with papers).  
```

Comment:
nit:  while we do have three co-chairs, when we refer to the singular role, we simplify to "Chair"

Reply:
```suggestion
Teams must have a Chair sponsor to maintain visibility in the team's work
and to ensure it is progressing.  The Chair sponsor also provides review
and approval as part of the process whether be on engagement or deliverable
publishing (such as with papers).  
```